impeller/renderer/backend/vulkan/swapchain_impl_vk.cc

```diff
@@ -470,10 +487,6 @@ bool SwapchainImplVK::Present(const std::shared_ptr<SwapchainImageVK>& image,
             // complete successfully.
             [[fallthrough]];
           case vk::Result::eSuccess:
-            is_rotated_ = false;
-            return;
-          case vk::Result::eSuboptimalKHR:
```

Comment:
We still need to treat this case as a success otherwise we'll exit from the application with validations enabled.

Reply:
Done!